pkg/ingester/ingester.go

```diff
@@ -81,8 +86,17 @@ type Config struct {
 	RejectOldSamples       bool
 	RejectOldSamplesMaxAge time.Duration
 
-	// For unit testing.
+	// maximum length settings for label names and values
+	MaxLengthLabelName  int
+	MaxLengthLabelValue int
+
+	// For testing, you can override the address and ID of this ingester
+	addr                  string
+	infName               string
+	id                    string
+	skipUnregister        bool
 	ingesterClientFactory func(addr string, cfg client.Config) (client.IngesterClient, error)
+	KVClient              ring.KVClient
```

Comment:
This smell like a bad merge or rebase to me, @khaine?

Reply:
certainly does... I'll try tidy this today